README.md

```diff
@@ -19,18 +19,34 @@ unacceptable behavior to [info@realm.io](mailto:info@realm.io).
 
 ## Installation
 
-Using [Homebrew](http://brew.sh/):
+### Using [Homebrew](http://brew.sh/):
 
 ```
 brew install swiftlint
 ```
 
+### Using [CocoaPods](https://cocoapods.org):
+
+Simply add the following line to your Podfile:
+
+```ruby
+pod 'SwiftLint'
+```
+
+This will download the SwiftLint binaries and dependencies in Pods/ during your next 
+`pod install` execution and will allow you to invoke it via `${PODS_ROOT}/SwiftLint/swiftlint` 
+in your Script Build Phases.
```

Comment:
Perhaps we should also say the following two things:

1. That this is the preferred installation method to install a specific version of SwiftLint to be used, since it supports installing a pinned version rather than simply the latest (which is the case with Homebrew).
2. This will add the SwiftLint binaries, its dependencies' binaries and the Swift binary library distribution to the `Pods/` directory, so checking in this directory to SCM such as git is discouraged.